core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -247,6 +247,9 @@ public TableWriter(
     ) {
         LOG.info().$("open '").utf8(tableToken.getTableName()).$('\'').$();
         this.configuration = configuration;
```

Comment:
It'd be good to prevent user from creating tables with system prefix (not necessarily in this PR).

Reply:
100% agree... and yes, probably should be a separate PR.

Reply:
Marking this one as resolved.